pkg/operator/sharded_secret.go

```diff
@@ -60,11 +60,7 @@ func (s *ShardedSecret) shard() []*v1.Secret {
 	s.secretShards = []*v1.Secret{}
 
 	// Ensure that we always iterate over the keys in the same order.
```

Comment:
we can probably remove the comment since the function name is explicit. We also don't need the intermediate `keys` variable.

Reply:
Thanks. Remove the comment done.